pyodata/v2/model.py

```diff
@@ -2072,13 +2083,24 @@ def parse(metadata_xml):
             mdf = io.BytesIO(metadata_xml)
         else:
             raise TypeError('Expected bytes or str type on metadata_xml, got : {0}'.format(type(metadata_xml)))
-        # the first child element has name 'Edmx'
-        edmx = etree.parse(mdf)
+
+        edmx: etree._ElementTree = etree.parse(mdf)
+
+        for node in edmx.iter():
+            qname = etree.QName(node.tag)
+
+            if qname.localname == 'Edmx' and qname.namespace in EDMX_WHITELIST:
+                NAMESPACES['edmx'] = qname.namespace
```

Comment:
Please, log a warning if the namespace is not on the whitelist.

Reply:
done